setup.py

```diff
@@ -21,7 +21,7 @@ def get_requirements():
 
 setup(
     name="pushsource",
-    version="1.1.0",
+    version="1.2.0",
```

Comment:
Should this be here? If yes then shouldn't it also be in the changelog?

Reply:
I did intentionally put this here, but there isn't really a well-established precedent for how to handle it.

These projects are meant to follow SemVer, so adding an API (like I've done here) means that the next version number should have the middle number bumped.  So: that bump can be included in the PR which adds the new API, which makes it the responsibility of the developer making a change to evaluate what it means with respect to SemVer.  Or, the bump can be *not* included in the PR, in which case whoever does the next release needs to inspect all the changes and figure out which of the three version numbers needs to be bumped per SemVer rules..

I think it's better to have the bump included in the PR because to me it seems like there's a high chance of release masters making a mistake when deciding the next version or even just forgetting to think about SemVer altogether.  I would like the whole process to be improved somehow though.

About whether it should be in the changelog, no I don't think so... I mean, there are arguments for and against, but 1.2.0 is still unreleased so it does make sense to me that the changelog still just says "Unreleased".  Note the 1.2.0 isn't set in stone... for example someone else could make a PR with backwards-incompatible changes meaning the next version should actually be 2.0.0... and it's probably more confusing to have a version number appear in a changelog, then disappear later, than it is to just group all unreleased content under "Unreleased".